frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -298,6 +298,53 @@ If you'd like to change the order in which dictionaries are queried (and their r
             }
         }
     }
+    if Device:canExternalDictLookup() then
+        local function genExternalDictItems()
+            local items_table = {}
+            for i, v in ipairs(Device:getExternalDictLookupList()) do
+                table.insert(items_table, {
```

Comment:
Some people object to what they call useless variables, but I'm of the opinion that something like this is easier to comprehend at a glance and therefore preferable:

```lua
local setting = v[1]
local text = v[2]
local enabled = v[3]
```